packages/building-blocks/package.json

```diff
@@ -27,5 +28,8 @@
   "bugs": {
     "url": "https://github.com/opentripplanner/otp-ui/issues"
   },
-  "gitHead": "0af1b7cda60bd4252b219dcf893e01c2acb2ed5d"
+  "gitHead": "0af1b7cda60bd4252b219dcf893e01c2acb2ed5d",
+  "dependencies": {
+    "styled-icons": "^10.47.1"
```

Comment:
This is causing a huge `yarn.lock` increase. We should probably only depend on the one icon set we need